src/components/ec-radio-btn/ec-radio-btn.spec.ts

```diff
@@ -0,0 +1,301 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from '@vue/test-utils';
+
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { EcRadioBtn } from '../../main';
+import type { RadioButtonProps } from './types';
+
+function mountRadioBtn(props?: Partial<RadioButtonProps>, mountOpts?: MountingOptions<RadioButtonProps>): CVueWrapper {
+  return mount(
+    EcRadioBtn as any, // eslint-disable-line
+    {
+      props: {
+        value: 'y',
+        label: 'Yes',
+        ...props,
+      },
+      ...mountOpts,
+    },
+  ) as CVueWrapper;
+}
+
+describe('EcRadioBtn', () => {
+  it('should render properly', () => {
+    const wrapper = mountRadioBtn();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    describe('name', () => {
+      it('should have a "name" attribute on the input equal to the "for" attribute on the label', () => {
+        const wrapper = mountRadioBtn();
+        const nameAttr = wrapper.findByDataTest('ec-radio-btn__input').attributes('name');
+        const forAttr = wrapper.findByDataTest('ec-radio-btn__label').attributes('for');
+        expect(nameAttr).toBe(forAttr);
+        expect(wrapper.findByDataTest('ec-radio-btn')).toMatchSnapshot();
+      });
+    });
+
+    describe('label', () => {
+      it('should render the radio button with the given label', () => {
+        const wrapper = mountRadioBtn();
+
+        expect(wrapper.findByDataTest('ec-radio-btn__label').exists()).toBe(true);
+        expect(wrapper.element).toMatchSnapshot();
+      });
+
+      it('should render the radio button without label when not present', () => {
+        const wrapper = mountRadioBtn({ value: 'y', label: undefined });
+
+        expect(wrapper.findByDataTest('ec-radio-btn__label').exists()).toBe(false);
+        expect(wrapper.element).toMatchSnapshot();
+      });
+    });
+
+    describe('isTextInline', () => {
+      it('should render the radio buttons text (label and description) on a single line', () => {
+        const wrapper = mountRadioBtn({
+          isTextInline: true,
+        });
+
+        expect(wrapper.findByDataTest('ec-radio-btn__text-wrapper').element).toMatchSnapshot();
```

Comment:
how it checks that label and description on the one line?

Reply:
by the css class passed